packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -142,11 +173,11 @@ class AbstractModal extends React.Component {
     } = this.props;
 
     if (!isOpen) {
-      this.toggleVisuallyHiddenMainDocument('false');
       return null;
     }
-    this.toggleVisuallyHiddenMainDocument('true');
 
+    // Store element that was last focused when modal was opened
+    this.modalTrigger = document.activeElement;
```

Comment:
It seems like the modalTrigger could get set to an incorrect value if the modal is updated (due to a async state change or context update) after its already opened. We don't have a lot of good lifecycle options to work with though. Maybe getDerivedStateFromProps when going from closed to open if we want to throw the modalTrigger in state?

Reply:
Updated this to store the modal trigger value in state right before we show the modal rather than in render: https://github.com/cerner/terra-framework/pull/608/commits/17c233a25469312c60e7e843d830840464ba14c9